implementations/rust/ockam/ockam_node/src/relay/processor_relay.rs

```diff
@@ -52,14 +52,15 @@ where
             Result::<()>::Ok(())
         };
 
+        #[allow(dead_code)] // TODO @antoinevg
```

Comment:
Can we fix it now?

Reply:
I feature-flagged it so It's fixed-ish enough for the embedded tech preview. Proper fix has to wait until either upstream stabilizes `select!()` for no_std or I implement our own.

Reply:
@antoinevg can you add those details to TODO?